models/Spaces.js

```diff
@@ -0,0 +1,32 @@
+import mongoose from 'mongoose';
+import { User } from './User';
+
+const ObjectId = mongoose.Schema.Types.ObjectId;
+const SpaceSchema = mongoose.Schema(
+  {
+    name: {
+      type: String,
+      required: true,
+      trim: true,
+    },
+    description: {
+      type: String,
+      required: true,
+      trim: true,
+    },
+    spaceId: {
+      type: String,
+      required: true,
+    },
+    isActive: {
+      type: Boolean,
+      required: true,
+    },
+    participants: [ObjectId],
```

Comment:
I believe ObjectId are used to refer to other modals within our database. Not sure how useful this would be without the `ref` attribute. This also leads to the issue of not being able to refer to anonymous users (users without accounts) who are in our spaces.

I think it's better to use an array of this type: https://mongoosejs.com/docs/schematypes.html#mixed
Make sure you remember this part though when making updates in the future 
```
Since Mixed is a schema-less type, you can change the value to anything else you like, but Mongoose loses the ability to auto detect and save those changes. To tell Mongoose that the value of a Mixed type has changed, you need to call doc.markModified(path), passing the path to the Mixed type you just changed.
```